.rubocop.yml

```diff
@@ -1596,19 +1593,17 @@ Style/NestedTernaryOperator:
 Style/Next:
   Enabled: false
 
-# Disabling for now
 Style/NilComparison:
-  Enabled: false
-  # EnforcedStyle: predicate
+  Enabled: true
+  EnforcedStyle: predicate
 
 # Disabling for now
 Style/NilLambda:
   Enabled: false
 
-# Disabling for now
 Style/NonNilCheck:
-  Enabled: false
-  # IncludeSemanticChanges: false
+  Enabled: true
+  IncludeSemanticChanges: false
```

Comment:
This cop's config is now redundant with [these defaults](https://github.com/rubocop/rubocop/blob/master/config/default.yml#L4383) and can be removed entirely from .rubocop.yml.

Reply:
Nice catch! Resolved by: 347dcbc919d0c1d5ad64891dee9e39346af654aa